app/services/public_interfaces/destroyer.rb

```diff
@@ -2,14 +2,28 @@ module PublicInterfaces
   class Destroyer
     include Serviceable
 
-    DestructionError = Class.new(StandardError)
-
     def call(environment)
+      @environment = environment
+      @service = "#{environment.app_id}-#{environment.id}-public" # Yes... it uses environment.id here and environment.name everywhere else! ¯\_(ツ)_/¯
```

Comment:
> `# Yes... it uses environment.id here and environment.name everywhere else! ¯\_(ツ)_/¯`

I don't think this comment is necessary or helpful. If you insist on identifying that `@service` is relying on `environment.id` because it isn't obvious, please decorate the `call` function itself.